frontend/components/sections/BitTableSection.tsx

```diff
@@ -85,18 +64,28 @@ export function BitTableSection({
    );
 }
 
-function Bit({ type, driverSizeColor, sizes }: GroupedBit) {
+function Bit({
+   groupedBit,
+   driverSizeColors,
+}: {
+   groupedBit: GroupedBit;
+   driverSizeColors: Record<string, string>;
+}) {
+   const { type, sizes, totalBitCount } = groupedBit;
    return (
       <Flex
+         display="inline-flex"
```

Comment:
CSS Grid is actually better-supported in terms of absolute numbers than `inline-flex` ([97.79%](https://caniuse.com/css-grid) rather than [97.38%](https://caniuse.com/mdn-css_properties_display_inline-flex)); would this layout be easier to implement with CSS Grid? (This may be a stupid question; I've not loaded the page to check what the layout actually looks like.)

Reply:
It is not necessary to have this element inline. This PR was discussed and refactored several times before shipping it, so it was probably a typo from one of the previous iterations